### DISCUSSION TITLE
What is the plan for components implemented in @fluentui/react package?

### DISCUSSION THREAD

**Discussion by sysofwan** [microsoft/fluentui#18540] [General] [answered]
From this [doc](https://github.com/microsoft/fluentui/wiki/Repo-structure#relevant-published-packages), looks like `@fluentui/react` package is meant as a "suite package" for re-exports. Are there plans to refactor components in `@fluentui/react` into individual packages?

We prefer consuming `@fluentui/react-{component}` packages to update components individually. It will also make it easier to transition from v7 -> v8.

**Comment by micahgodbolt** [comment, marked as answer]
the /react package is going to stay as is. @fluentui/react-components is now our 'suite package' for re-export of our new v9 controls.
